tests/test_run.py

```diff
@@ -71,3 +71,13 @@ def test_run__wrong_input_raises_error(mocker, extra_model_mock, pandas_mock):
 
     with pytest.raises(ExtraModelError, match="wrong_column_1"):
         run(mocker.MagicMock(), mocker.MagicMock())
+
+
+def test_run__run_from_dataframe(mocker, extra_model_mock, pandas_mock):
```

Comment:
can you also add one more test that passes wrong `df` to make sure we trigger the error? There is an example of it already above this test.

Reply:
Added, run was using run_from_dataframe so that is why it showed 100% coverage even before